pkg/runner/cluster_k8s.go

```diff
@@ -496,17 +495,16 @@ func (c *ClusterK8sRunner) CollectOutputs(ctx context.Context, input *api.Collec
 		VersionedParams(&v1.PodExecOptions{
 			Container: "collect-outputs",
 			Command: []string{
-				"tar",
-				"-czf",
-				"-",
-				outputPath,
+				"sh",
```

Comment:
We can avoid the extra shell
`tar -C /outputs -czf - input.RunID`

Reply:
Fixed.